scripts/create_release.py

```diff
@@ -20,26 +21,36 @@ def version_key(version_string):
   return key
 
 
-def main(args):
+def main():
   if subprocess.check_output(['git', 'status', '--porcelain'], cwd=root_dir).strip():
     print('tree is not clean')
     sys.exit(1)
 
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-r', '--release-hash')
+  parser.add_argument('-a', '--asserts-hash')
+  parser.add_argument('-v', '--new-version')
+  parser.add_argument('--action', action='store_true')
+  options = parser.parse_args()
+
   release_info = emsdk.load_releases_info()
-  new_version = version_key(release_info['aliases']['latest'])[0]
-  new_version[-1] += 1
+  if options.new_version:
+    new_version = options.new_version
+  else:
+    new_version = version_key(release_info['aliases']['latest'])[0]
+    new_version[-1] += 1
+    new_version = '.'.join(str(part) for part in new_version)
 
-  new_version = '.'.join(str(part) for part in new_version)
   asserts_hash = None
   is_github_runner = False
-  if args:
-    new_hash = args[0]
-    if len(args) > 1:
-      asserts_hash = args[1]
-    if len(args) > 2 and args[2] == '--action':
-      is_github_runner = True
+  if options.release_hash:
+    new_hash = options.release_hash
+    asserts_hash = options.asserts_hash
   else:
     new_hash = emsdk.get_emscripten_releases_tot()
+  if options.action:
+      is_github_runner = True
```

Comment:
Can we just remove `is_github_runner` and use `options.gh_action` below instead?